lib/sessionaccount.js

```diff
@@ -129,6 +129,26 @@ class SessionAccount {
 
   }
 
+  doCheckLoginIfNeeded(idToken) {
+    //We have the idToken in a cookie, check if we have performed a check login yet, indicator stored in cookie hasCheckedLoginUserId
+    const userId = Cookies.get("session.hasCheckedLoginUserId");
+    if (!userId) {
+      const completeCheckUserIDPromise = this.performCheckLoginToServer(idToken);
+
+      const promiseToReturn = completeCheckUserIDPromise.then(() => {
+        const validatedUserId = UserInfoStash.getUserId();
+        Cookies.set("session.hasCheckedLoginUserId", validatedUserId,  { domain: this.domainName, secure: true  });
+        return Promise.resolve(idToken);
```

Comment:
```suggestion
        return idToken;
```
should be equivalent